plugins/panels/similarity_search/__init__.py

```
@@ -293,15 +294,24 @@ def on_change_view(self, ctx):
+
         sample_ids = ctx.params.get("sample_ids", [])
         if not sample_ids:
             return
 
         try:
             view = ctx.dataset.select(sample_ids)
-            filepaths = dict(zip(view.values("id"), view.values("filepath")))
-            ctx.panel.set_data("sample_media", filepaths)
+            ids, filepaths = view.values(["id", "filepath"])
+            media = {
+                sid: get_signed_url(fp) for sid, fp in zip(ids, filepaths)
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)